content/docs/visualization/consoles.md

```diff
@@ -25,15 +28,142 @@ Consoles have 5 parts:
 1. The main content in the center, usually graphs
 1. A table on the right
 
-The navigation bar is for links to other systems, such as other Prometheuses, documentation and whatever else makes sense to you.
-The menu is for navigation inside the Prometheus, it's very useful to be able to quickly open a console in another tab to correlate information.
-Both are configured in `console_libraries/menu.lib`.
+The navigation bar is for links to other systems, such as other Prometheuses,
+documentation and whatever else makes sense to you. The menu is for navigation
+inside the Prometheus, it is very useful to be able to quickly open a console
```

Comment:
run-on-sentence can be fixed by s/it/which/

"the Prometheus" -> "the same Prometheus server"?